### PR TITLE
2.0 logs

### PR DESCRIPTION
**Pull request by ngilain**
[OO-FEATURE] Add log screen
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2065
https://github.com/open-orchestra/open-orchestra-libs/pull/73
https://github.com/open-orchestra/open-orchestra/pull/1130

### CODE REVIEW

File: LogBundle/Resources/public/ecmascript/OpenOrchestra/Application/Model/Log/Log.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+     * Pre initialize
+     */
+    preinitialize() {
+        this.idAttribute = 'id';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    _getSyncUrl(method, options) {
```
Comment (alavieille) [inline LogBundle/Resources/public/ecmascript/OpenOrchestra/Application/Model/Log/Log.js]: this method is useless

File: LogBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Log/LogsView.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     */
+    render() {
+        let template = this._renderTemplate('Log/logsView');
+        this.$el.html(template);
+        this._listView = new LogListView({
+            collection: this._collection,
+            settings: this._settings
+        });
+        $('.logs-list', this.$el).html(this._listView.render().$el);
+        $('.datepicker', this.$el).datepicker();
```
Comment (alavieille) [inline LogBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Log/LogsView.js]: if you not sepecify region the datepicker is translated ?

File: ModelLogBundle/Document/Log.php [hunk 3]
```diff
@@ -73,6 +73,7 @@ class Log implements LogInterface
      * @ORCHESTRA\Search(field="extra.user_ip",key="user_ip")
      * @ORCHESTRA\Search(field="extra.user_name",key="user_name")
      * @ORCHESTRA\Search(field="extra.site_name",key="site_name")
+     * @ORCHESTRA\Search(field="extra.site_id",key="site_id")
```
Comment (alavieille) [inline ModelLogBundle/Document/Log.php]: remove annotation search is useless now

File: ModelLogBundle/Repository/LogRepository.php [hunk 4]
```diff
@@ -3,13 +3,87 @@
...
  */
 class LogRepository extends AbstractAggregateRepository implements LogRepositoryInterface
 {
-    use PaginationTrait;
+    /**
+     * @param PaginateFinderConfiguration $configuration
+     *
+     * @return array
+     */
+    public function findForPaginate(PaginateFinderConfiguration $configuration)
```
Comment (alavieille) [inline ModelLogBundle/Repository/LogRepository.php]: add functional test for this method

File: ModelLogBundle/Repository/LogRepository.php [hunk 5]
```diff
@@ -3,13 +3,87 @@
...
+        $qa->skip($configuration->getSkip());
+        $qa->limit($configuration->getLimit());
+
+        return $this->hydrateAggregateQuery($qa);
+    }
+
+    /**
+     * @return int
+     */
+    public function count()
```
Comment (alavieille) [inline ModelLogBundle/Repository/LogRepository.php]: add functional test for this method

File: ModelLogBundle/Repository/LogRepository.php [hunk 6]
```diff
@@ -3,13 +3,87 @@
...
+
+        return $this->countDocumentAggregateQuery($qa);
+    }
+
+    /**
+     * @param PaginateFinderConfiguration $configuration
+     *
+     * @return int
+     */
+    public function countWithFilter(PaginateFinderConfiguration $configuration)
```
Comment (alavieille) [inline ModelLogBundle/Repository/LogRepository.php]: add functional test for this method

**Comment by ngilain** [comment]
@alavieille : update done

**Comment by alavieille** [comment]
:+1:

**Pull request merged**